src/content/cookbook/testing/integration/introduction.md

```diff
@@ -27,88 +27,104 @@ This recipe uses the following steps:
-
-## 2. Add the `integration_test` dependency
-
-Run following command to add `integration_test` and `flutter_test` packages as `dev_dependencies` using `sdk: flutter`.
+## Create a new app to test
+
+Integration testing requires an app to test.
+This example uses the built-in **Counter App** example
+that Flutter produces when you run the `flutter create` command.
+The counter app allows a user to tap on a button to increase a counter.
+
+1. To create an instance of the built-in Flutter app,
+   run the following command in your terminal:
+
+   ```console
+   $ flutter create counter_app
+   ```
+
+1. Change into the `counter_app` directory.
+
+1. Open `lib/main.dart` in your preferred IDE.
+   It should resemble the following code:
+
+   <?code-excerpt "lib/main.dart"?>
+   ```dart title="lib/main.dart"
+   import 'package:flutter/material.dart';
+
+   void main() => runApp(const MyApp());
+
+   class MyApp extends StatelessWidget {
+     const MyApp({super.key});
+
+     @override
+     Widget build(BuildContext context) {
+       return const MaterialApp(
+         title: 'Counter App',
+         home: MyHomePage(title: 'Counter App Home Page'),
+       );
+     }
+   }
+
+   class MyHomePage extends StatefulWidget {
+     const MyHomePage({super.key, required this.title});
+
+     final String title;
+
+     @override
+     State<MyHomePage> createState() => _MyHomePageState();
+   }
+
+   class _MyHomePageState extends State<MyHomePage> {
+     int _counter = 0;
+
+     void _incrementCounter() {
+       setState(() {
+         _counter++;
+       });
+     }
+
+     @override
+     Widget build(BuildContext context) {
+       return Scaffold(
+         appBar: AppBar(
+           title: Text(widget.title),
+         ),
+         body: Center(
+           child: Column(
+             mainAxisAlignment: MainAxisAlignment.center,
+             children: <Widget>[
+               const Text(
+                 'You have pushed the button this many times:',
+               ),
+               Text(
+                 '$_counter',
+                 style: Theme.of(context).textTheme.headlineMedium,
+               ),
+             ],
+           ),
+         ),
+         floatingActionButton: FloatingActionButton(
+           // Provide a Key to this button. This allows finding this
+           // specific button inside the test suite, and tapping it.
+           key: const Key('increment'),
+           onPressed: _incrementCounter,
+           tooltip: 'Increment',
+           child: const Icon(Icons.add),
+         ),
+       );
+     }
+   }
+   ```
+
+## Add the `integration_test` dependency
+
+You need to add the testing packages to your new app.
+
+To add `integration_test` and `flutter_test` packages as
+`dev_dependencies` using `sdk: flutter`, run following command.
 
-```sh
-$ flutter pub add 'dev:flutter_test:{"sdk":"flutter"}'  'dev:integration_test:{"sdk":"flutter"}'
+```console
+$ flutter pub add 'dev:flutter_test:{"sdk":"flutter"}' \
```

Comment:
Isn’t this already included by the default template? If you want this command to do something, creating an app using the Flutter empty template would make more sense?